FlowCrypt/Controllers/MessageList Extension/MsgListViewController.swift

```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+@MainActor
```

Comment:
Each protocol intended to work with UI state should be marked with `@MainActor` to be sure that we work with it on UI thread.